portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Megha!</h1>");
+    response.getWriter().println("Kiwis contain more Vitamin C than oranges :o");
```

Comment:
I don't think so, I think it was there before because there was no html file connected to it. If I include it now, the text on the page shows the <h1> and </h1> tags as well